scripts/cnode-helper-scripts/blockPerf.sh

```diff
@@ -91,8 +93,9 @@ fi
 #Deploy systemd if -d argument was specified
 if [[ "${DEPLOY_SYSTEMD}" == "Y" ]]; then
   # if not already enabled activate the required Tracers in the config file
-  [[ "$(jq -r .TraceChainSyncClient "${CONFIG}")" != "true" ]] && jq '.TraceChainSyncClient = true' ${CONFIG} > ${CONFIG}.tmp && mv ${CONFIG}.tmp ${CONFIG} && echo "INFO: Enabling node config TraceChainSyncClient" && config_changed=1
-  [[ "$(jq -r .TraceBlockFetchClient "${CONFIG}")" != "true" ]] && jq '.TraceBlockFetchClient = true' ${CONFIG} > ${CONFIG}.tmp && mv ${CONFIG}.tmp ${CONFIG} && echo "INFO: Enabling node config TraceBlockFetchClient" && config_changed=1
+  [[ "$(jq -r .TraceChainSyncClient "${CONFIG}")" != "true" ]] && jq '.TraceChainSyncClient = "true"' ${CONFIG} > ${CONFIG}.tmp && mv ${CONFIG}.tmp ${CONFIG} && echo "INFO: Enabling node config TraceChainSyncClient" && config_changed=1
+  [[ "$(jq -r .TraceBlockFetchClient "${CONFIG}")" != "true" ]] && jq '.TraceBlockFetchClient = "true"' ${CONFIG} > ${CONFIG}.tmp && mv ${CONFIG}.tmp ${CONFIG} && echo "INFO: Enabling node config TraceBlockFetchClient" && config_changed=1
+  [[ "$(jq -r .TracingVerbosity "${CONFIG}")" != "NormalVerbosity" ]] && jq '.TracingVerbosity = "NormalVerbosity"' ${CONFIG} > ${CONFIG}.tmp && mv ${CONFIG}.tmp ${CONFIG} && echo "INFO: Set node config NormalVerbosity" && config_changed=1
```

Comment:
6 jq calls can be replaced with 1, see example [here](https://github.com/cardano-community/guild-operators/blob/setup-upds/scripts/grest-helper-scripts/setup-grest.sh#L302-L314), which can then be used as array (eg: [here](https://github.com/cardano-community/guild-operators/blob/setup-upds/scripts/grest-helper-scripts/setup-grest.sh#L290-L295)